### PR TITLE
Mark classes as final

### PR DESCRIPTION
**Pull request by keradus**
Hi!
First of all, it is going to be a great lib!

Then, about PR - shouldn't internal classes be final where it is possible?

### CODE REVIEW

File: tests/Intl/Grapheme/GraphemeTest.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
  *
  * @covers Symfony\Polyfill\Intl\Grapheme\Grapheme::<!public>
  */
-class GraphemeTest extends \PHPUnit_Framework_TestCase
+final class GraphemeTest extends \PHPUnit_Framework_TestCase
```
Comment (stof) [inline tests/Intl/Grapheme/GraphemeTest.php]: marking tests as final is not really useful

Comment (nicolas-grekas) [inline tests/Intl/Grapheme/GraphemeTest.php]: can you remove the final keywork on tests @keradus please?

Comment (keradus) [inline tests/Intl/Grapheme/GraphemeTest.php]: OK, I wasn't sure so I mark them in second commit on purpose ;)

done

**Comment by nicolas-grekas** [comment]
Thank you @keradus.

**Pull request merged**